src/app/components/data-entry-grid/index.html

```diff
@@ -117,6 +117,11 @@ <h2>ag-Grid wrapper component</h2>
         The <sky-code>sky-ag-grid-wrapper</sky-code> component provides WCAG-compliant keyboard navigation and sticky column headers for grids that use auto height. Use the <sky-code>sky-ag-grid-wrapper</sky-code>  component to wrap all instances of <sky-code>ag-grid-angular</sky-code>. The wrapper does not constrain standard ag-Grid functionality.
       </p>
 
+      <h2>Data manager directive</h2>
+      <p>
+        If you're using an ag-Grid within the <a routerLink="../../../skyux-data-manager/docs/data-manager">data manager component</a> there is a directive <sky-code>skyAgGridDataManagerAdapter</sky-code> that will manage some of the interactions between the grid and the data manager service that are typically the same across grid uses. The directive can be added to the <sky-code>sky-data-view</sky-code> element containing the grid. It will initialize and keep up to date the columns listed as visible in the data state, update the data state when the columns are moved or changed within the grid, initialize and keep up to date the selected rows from the data state, update the data state when row selection changes within the grid, and update the data state when the sort is changed within the grid.
```

Comment:
In some places, we just refer to the grid as a grid and not _ag-Grid_ so a question I have for the first sentence is should we swap out "ag-Grid" for "grid?"

I'd probably tweak the first sentence a bit too to, "If you use a grid within the data manager component, you can manage similar interactions between the grid and data manager service with the skyAgGridDataManagerAdapter directive." 

I would want to start that second sentence by saying, "`sky-data-view-elements` containing grids can also use the directive," but not sure if we try to stray from starting sentences with code snippets like that. I'd defer to @blackbaud-johnly on that.

Last sentence is kinda long, and I'm not sure what the "It" is at the beginning...is it the directive? If yes, I'd lean on a bulleted list and say something like, 
"The directive updates and initializes:
- columns listed in the data state
- data states when columns move or change within the grid
- selected rows from the data state..."

and so on. Again, will let John check me on that.

Reply:
I think it should definitely be ag-grid here, just to make sure it is clear that this doesn't work with the skyux-grid.

Similar interactions doesn't quite explain what it does I don't think. Basically the way sort or search or filters are updated is the exact same across 99% of data manager ag grid usages. To save them from having to repeat code constantly, they can use this directive. Maybe standard interactions instead of similar?

Also why "sky-data-view-elements containing grids can also use the directive"? They're the only element that the directive can be placed on. Does it sound like it goes on the ag-grid currently?

The bulleted list sounds like a good idea.

Reply:
I played around with this some, trying to take into account the feedback from Denise and Alex and also trying to tighten up a little. I also changed all instances of `grid` to `data entry grid` to make sure we're clear. What do you think of this version with my suggested updates in place?

```If you use a data entry grid in a data manager component, the skyAgGridDataManagerAdapter directive can manage some standard interactions between the data entry grid and the data manager service. Add the directive to the sky-data-view element that contains the data entry grid to initialize properties from the data state and keep the data entry grid in sync with the data state. When the data entry grid changes, the data state is updated, and when the data state changes, the data entry grid responds to the changes. The directive manages:
•	column visibility and order
•	selected rows
•	active sort state when selected by column header
Other properties of the data state, such as filters and applying the sort, still need to be implemented for each use.
```

Reply:
Resolving this conversation. Erika and I talked offline, and I updated the text.